src/sentry/static/sentry/app/views/organizationIntegrations/providerRow.tsx

```diff
@@ -125,19 +124,19 @@ export default class ProviderRow extends React.Component<Props> {
   render() {
     return (
       <PanelItem p={0} flexDirection="column" data-test-id={this.props.provider.key}>
```

Comment:
Can we also refactor this into a styled component? (Probably belongs in a new PR)